x/ethbridge/types/msgs.go

```diff
@@ -36,6 +37,12 @@ func (msg MsgCreateEthBridgeClaim) ValidateBasic() sdk.Error {
 	if !gethCommon.IsHexAddress(msg.EthereumSender.String()) {
 		return ErrInvalidEthAddress(DefaultCodespace)
 	}
+	if !gethCommon.IsHexAddress(msg.BridgeContractAddress.String()) {
+		return ErrInvalidEthAddress(DefaultCodespace)
+	}
+	if strings.ToLower(msg.Symbol) == "eth" && msg.TokenContractAddress != NewEthereumAddress("0x0000000000000000000000000000000000000000") {
```

Comment:
use `!IsZeroAddress(msg.TokenContractAddress)`

Reply:
Was not able to implement this requested change due to a typing error, which I've created an issue for https://github.com/cosmos/peggy/issues/72. I'll address this asap, but the issue does not block this PR.